content/spin/v2/variables.md

```diff
@@ -117,28 +128,31 @@ fn handle_spin_example(req: Request) -> Result<impl IntoResponse> {
 
 {{ startTab "TypeScript"}}
 
-> [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-js-sdk/variables/Config.html)
+> [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-js-sdk/modules/Variables.html)
 
 > Note that the name is `Config` rather than `Variables`.
 
 ```ts
 import { HandleRequest, HttpRequest, HttpResponse, Config } from "@fermyon/spin-sdk"
 
-const decoder = new TextDecoder("utf-8")
-
 export const handleRequest: HandleRequest = async function (request: HttpRequest): Promise<HttpResponse> {
-  const expected = decoder.decode(request.body)
-  let password = Config.get("password")
-  let access = "denied"
-  if (expected === password) {
-      access = "accepted"
-  }
-  let responseJson = `{\"authentication\": \"${access}\"}`;
-
+  let token = Config.get("token")
```

Comment:
Heads up that this is changing in JS SDK v2 and there is #1355 in progress for that, so some synchronisation might be needed...

Reply:
Rebased!